src/components/organisms/AppRouter/AppRouter.tsx

```diff
@@ -97,15 +119,15 @@ export const AppRouter: React.FC = () => {
 
           <Route
             path="/sparkleverse"
-            render={() => {
+            component={() => {
               window.location.href = SPARKLEVERSE_HOMEPAGE_URL;
               return <LoadingPage />;
             }}
           />
 
           <Route
             path="/"
-            render={() => {
+            component={() => {
               window.location.href = DEFAULT_REDIRECT_URL;
               return <LoadingPage />;
             }}
```

Comment:
Why were these changed? They were correct as the were before, and this change makes them broken and invalid.

Reply:
**Further Reading:**

- https://reactrouter.com/web/api/Route/route-render-methods
  - > **The recommended method of rendering something with a `<Route>` is to use children elements**, as shown above. There are, however, a few **other methods** you can use to render something with a `<Route>`. These are provided mostly for **supporting apps that were built with earlier versions of the router before hooks were introduced.**
  - https://reactrouter.com/web/api/Route/component
    - > When you use component (instead of render or children, below) the router uses `React.createElement` to create a new React element from the given component. That means **if you provide an inline function to the component prop, you would create a new component every render. This results in the existing component unmounting and the new component mounting instead of just updating the existing component.** When using an inline function for inline rendering, use the `render` or the `children` prop (below).
- https://reactrouter.com/web/api/Route/render-func
  - > This allows for convenient inline rendering and wrapping **without the undesired remounting explained above.**

Reply:
> Why were these changed? They were correct as the were before, and this change makes them broken and invalid.

Typo. I had converted multiple `component` to `render`, but reverted them. This one shouldn't have.

Reply:
Reverted 1ba4f769cbdd73764c103e8cde1ba2a26f33f772